code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -237,4 +240,22 @@ protected Logger getLogger() {
     public String getName() {
         return getConstantsConfig().getProviderName();
     }
+
+    public String validateIdentities(List<Map<String, String>> identitiesGiven) {
+        List<Identity> identities = getIdentities(identitiesGiven);
+        return openLDAPUtils.toHashSeparatedString(identities);
+    }
```

Comment:
The above method validates each identity using ID and type set in the API against the openLDAP backend. Returns a '#' separated string of the identities.